app/cache/index.js

```diff
@@ -1,15 +1,12 @@
-const { start, stop, get, set, update, clear, flushAll } = require('./base')
-const getCachedResponse = require('./get-cached-response')
-const setCachedResponse = require('./set-cached-response')
+const { start, stop } = require('./base')
+const { getCachedResponse } = require('./get-cached-response')
+const { setCachedResponse } = require('./set-cached-response')
+const { getCacheKey } = require('./get-cache-key')
 
 module.exports = {
   start,
   stop,
-  get,
-  set,
-  update,
-  clear,
-  flushAll,
+  getCacheKey,
   getCachedResponse,
   setCachedResponse
```

Comment:
Maybe keep these in the order they are in at the top of the file - unless they have to be exported this way?